r/R/dplyr-funcs-datetime.R

```diff
@@ -140,3 +140,22 @@ register_bindings_datetime <- function() {
     (year %% 4 == 0) & ((year %% 100 != 0) | (year %% 400 == 0))
   })
 }
+
+binding_format_datetime <- function(x, format = "", tz = "", usetz = FALSE) {
+  if (usetz) {
+    format <- paste(format, "%Z")
+  }
+
+  if (call_binding("is.POSIXct", x)) {
+    if (tz == "" && x$type()$timezone() != "") {
+      tz <- x$type()$timezone()
+    } else if (tz == "") {
+      tz <- Sys.timezone()
+    }
```

Comment:
Would some of this stuff change when #12240 merges? If so, we should add a note here that we should revisit this behavior when that merges